two_factor_authentication.rst

```diff
@@ -0,0 +1,39 @@
+Zwei Faktor Authentifizierung
+========================
+
+Hitobito unterstützt die Zwei Faktor Authentifizierung mit folgenden Methoden:
+
+- TOTP mittels QR Code und sechsstelligem Code
+
+TOTP
+========================
+
+Voraussetzungen
+-------------------------------
+
+Für die zwei Faktor Authentifizierung wird ein Smartphone und die freeOTP App: https://freeotp.github.io/ benötigt. Zum Herunterladen der App benötigen Sie eine Internetverbindung und evtl. ein Konto bei Ihrem App Store. Die App ist komplett kostenfrei.
+
+Einrichten
+------------------------------
+
+Um die zwei Faktor Authentifizierung einzurichten, müssen Sie sich erst in Hitobito anmelden. Navigieren Sie anschliessend mittels ihrem Namen als Link oben rechts auf ihr Profil. Sie sollten nun unter dem Dropdown "Login" den Knopf "Zwei Faktor Authentifizierung einrichten" sehen. Sobald Sie den entsprechenden Knopf drücken, landen Sie auf einer neuen Seite mit einem QR Code.
```

Comment:
```suggestion
Um die Zwei-Faktor-Authentifizierung einzurichten, muss man sich erst in Hitobito anmelden. Auf dem eigenen Profil kann man nun im Dropdown "Login" den Knopf "Zwei Faktor Authentifizierung einrichten" sehen. Durch Klicken auf diesen Knopf landet man auf einer Seite mit einem QR-Code.
```
Der Rest der user documentation ist in Du-Form geschrieben. Ich würde vorschlagen, Du oder Sie zu vermeiden wo möglich.